### PR TITLE
Improve cancel command

### PR DESCRIPTION
**Pull request by noplanman**
Let the user know which conversation has been cancelled.
Add more documentation.
Refactor code and adjust output to user.

### CODE REVIEW

File: src/Commands/UserCommands/CancelCommand.php [hunk 1]
```diff
@@ -54,15 +60,24 @@ public function execute()
      */
     public function executeNoDB()
     {
-        //Database not setted or without connection
-        $message = $this->getMessage();
-        $chat_id = $message->getChat()->getId();
+        //Output the default message when there is no DB connection
+        parent::executeNoDB();
```
Comment (MBoretto) [inline src/Commands/UserCommands/CancelCommand.php:64]: Why I should say that db is not connected?

Comment (MBoretto) [inline src/Commands/UserCommands/CancelCommand.php]: Why I should say that db is not connected?

Comment (noplanman) [inline src/Commands/UserCommands/CancelCommand.php:64]: You don't :smiley: 

Feel free to change this and output any message.
I just set it to what other commands output when there is no DB connection.

You're right, it makes more sense to just output "No active conversation!" or something :+1:

Comment (MBoretto) [inline src/Commands/UserCommands/CancelCommand.php:64]: If you build a command with a keyboard you can have the possibility to clear the chat. 
Know that DB is not connected is excessive. Is just to let this command be as generic as possible!

**Pull request merged**